src/components/button/Button.js

```diff
@@ -381,5 +381,16 @@ export default class ButtonComponent extends BaseComponent {
   focus() {
     this.buttonElement.focus();
   }
-}
 
+  getView() {
+    if (this.component.displayInEditGrid) {
+      this.build();
+       setTimeout(() => {
+        const element = document.getElementById(this.id);
+        element.parentNode.replaceChild(this.element, element);
+      });
+       return this.element.outerHTML;
```

Comment:
This really seems strange to me. I think I may need you to show me tomorrow what this is doing.